madgiraffestudios/src/components/About/About.scss

```diff
@@ -2,17 +2,39 @@
 
 .description-text { width: 75%; }
 
+#photoshoots{ margin-top: 258px;}
```

Comment:
- [ ] let's have a look at this today, you shouldn't have to set special margin-top on each div element in the same container